english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/friendly-dragon.md

```diff
@@ -1,4 +1,10 @@
 A dragon the size of an elephant, sits in front of you and gazes adoringly at the apple.
 It wags its tail like a giant Golden Retriever.  It is wearing a saddle, so you feed it 
 the rest of the apple and climb aboard for a joyride.  It gallops along a long hallway,
-jumps out the window, and takes flight.
\ No newline at end of file
+jumps out the window, and takes flight.
+
+[Do you tell the dragon to stop immediately, because you're scared of heights?](stop-immediately.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.